packages/cli/src/runAndroid/runAndroid.js

```diff
@@ -191,6 +195,34 @@ function tryInstallAppOnDevice(args, device) {
   }
 }
 
+function getInstallApkName(appFolder, device, buildDirectory) {
+  const gradleFile = fs.readFileSync(`${appFolder}/build.gradle`, 'utf8');
+
+  const enableSeparateBuild = gradleFile.match(
+    /def enableSeparateBuildPerCPUArchitecture.*=.*(true|false)/i
```

Comment:
So this is the default variable name `enableSeparateBuildPerCPUArchitecture` from RN template. I think we should check for:

```
 abi {
   enable true|false
 }
```

The same way you are looking for `universalApk`.

Reply:
@ferrannp Good point. I think the probability is low, but it is possible to set the value immediately without `enableSeparateBuildPerCPUArchitecture`. I'm thinking of a different approach.